Rakefile

```diff
@@ -27,3 +27,9 @@ require "yard/rake/yardoc_task"
 
 YARD::Rake::YardocTask.new do |yard|
 end
+
+# RuboCop requires Ruby >= 2.2
+if Gem::Version.new(RUBY_VERSION) >= Gem::Version.new("2.2.0")
```

Comment:
Just saw I use `Gem.ruby_version` and `Gem::Version.new(RUBY_VERSION)` instead of using one consistently. Which version do you prefer? `Gem.ruby_version` already returns a `Gem::Version` which is nice.

Reply:
Oh not sure. I guess `Gem.ruby_version` is simpler.